### DISCUSSION TITLE
How do I enable execution time segment?

### DISCUSSION THREAD

**Discussion by AHandless** [JanDeDobbeleer/oh-my-posh#3941] [Troubleshoot] [answered]
PowerShell: 7.3.4
oh-my-posh version: 17.0.0
Windows Terminal version: 1.16.10261.0

Hello. I'm trying to get the execution time segment working, but it won't show up. I'm using the default config `jandedobbeleer.omp.json` According to the documentation:
> To use this feature, either utilize the PowerShell module or ensure that you pass an execution-time argument containing the elapsed milliseconds to the oh-my-posh executable. The installation guide provides instructions on how to include this argument for PowerShell and Zsh.

Unfortunately, there are no examples provided on how to pass this argument to oh-my-posh. oh-my-posh.exe says `Error: unknown flag: --execution-time`

**Comment by JanDeDobbeleer** [comment, marked as answer]
@AHandless that's not necessary, the docs are outdated there (by about 2 years so it's weird nobody noticed before). The default config isn't `jandedobbeleer.omp.json` unless you specified that one using the `--config` flag.

To enable this you'll need to export the config to a local file, edit it by adding the `executiontime` segment and reference you local config using the local file with the `--config` flag.

See [here](https://ohmyposh.dev/docs/installation/customize#adjust-a-theme).

**Reply by AHandless** [reply]
Thank you so much for your quick reply! I mistakenly specified the wrong path to the config in my $PROFILE, and there was no indication of the error. Consequently, I believed that the config was correct, but it turned out that the loaded config did not include the execution time segment.
